pkg/util/prometheus/parse.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
@adel121 These `pkg/util/prometheus` files need a new corresponding CODEOWNERS entry since `pkg/util` is owned by ASC

Reply:
Alright.

Thanks